CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# 4.4.0
+
+* Adds support for applications that use Minitest by adding an `AssertMatchers` module ([#66](https://github.com/alphagov/govuk_schemas/pull/66))
+* Moves validation and error construction into a `Validator` module
```

Comment:
This isn't needed for the changelog because it's an internal concern. Changelog only need contain information needed for users of the gem.

Reply:
have removed